site/content/api/Tenant-API.md

```diff
@@ -3,7 +3,10 @@ title = "Tenant API"
 weight = 410
 +++
 
-The *Tenant API* is used to make Hono aware of the tenants that are available in an installation. A tenant is a logical entity, which groups together a set of devices and consists of meta information which should be used by other Hono components. It is completely separated from other tenants, so devices can never belong to more than one tenant. The tenant meta information is used for example to determine if a given tenant is allowed to use a certain protocol adapter or if devices are required to use authentication. 
+The *Tenant API* is used to make Hono aware of the tenants that are available in an installation. A tenant is a logical entity, which groups together a set of devices and consists of meta information which should be used by other Hono components. It is completely separated from other tenants, so devices can never belong to more than one tenant. The tenant meta information is used for example to determine if a given tenant is allowed to use a certain protocol adapter or if devices are required to use authentication.
```

Comment:
can you please add a 
````
<!--more-->
````
after this paragraph to delimit the introductory paragraph.

Reply:
added